File: 151002/Bobrik/NewsApi/src/main/java/by/bsuir/newsapi/model/request/TagRequestTo.java
package by.bsuir.newsapi.model.request;

import jakarta.validation.constraints.NotBlank;
import org.hibernate.validator.constraints.Length;

public record TagRequestTo(
        Long id,
        @NotBlank
        @Length(min = 2, max = 32)
        String name) {
}

File: 151004/Gonzarevich/lab3/publisher/DTO/TagRequestTo.php
<?php

namespace App\DTO;

use Symfony\Component\Validator\Constraints as Assert;

class TagRequestTo
{
    public function __construct(

        public readonly ?int $id,

        #[Assert\NotBlank]
        #[Assert\Length(min: 2, max: 32)]
        public readonly string $name,
    ) {
    }
}